ReactWindows/ReactNative.Shared/UIManager/DragDropEvent.cs

```diff
@@ -0,0 +1,29 @@
+using Newtonsoft.Json.Linq;
+using ReactNative.UIManager.Events;
+
+namespace ReactNative.UIManager
+{
+    class DragDropEvent : Event
```

Comment:
>DragDropEvent [](start = 10, length = 13)

If this isn't supported in WPF - lets just leave this in the UWP project (instead of Shared)

Reply:
Fixed